src/main/java/net/sf/jabref/gui/util/IconComparator.java

```diff
@@ -18,16 +18,17 @@
 import net.sf.jabref.model.entry.BibtexEntry;
 
 import java.util.Comparator;
+import java.util.List;
 
 /**
  * Comparator that handles icon columns.
  */
 public class IconComparator implements Comparator<BibtexEntry> {
 
-    private final String[] fields;
+    private final List<String> fields;
 
 
-    public IconComparator(String[] fields) {
+    public IconComparator(List fields) {
```

Comment:
Yes. Interestingly, when you type in `List<String>` without the quotation marks, the `<String>` part is not shown, only when editing the post.